airtime_mvc/application/models/StreamSetting.php

```diff
@@ -464,7 +464,7 @@ public static function setDjLiveStreamPort($value)
 
     public static function getDjLiveStreamPort()
     {
-        return self::getValue("dj_live_stream_port", 8001);
+        return self::getValue("dj_live_stream_port", 8002);
```

Comment:
Yeah, I'm just going off the code requiring them to be different. NATing 8002 is a good idea.